debian/rules

```diff
@@ -56,9 +56,13 @@ override_dh_systemd_enable:
 	dh_systemd_enable -pubuntu-advantage-tools ua-reboot-cmds.service
 	dh_systemd_enable -pubuntu-advantage-tools ua-timer.timer
 	dh_systemd_enable -pubuntu-advantage-tools ua-timer.service
+	dh_systemd_enable -pubuntu-advantage-tools ua-license-check.timer
+	dh_systemd_enable -pubuntu-advantage-tools ua-license-check.service
+	dh_systemd_enable -pubuntu-advantage-tools ua-license-check.path
 
 override_dh_systemd_start:
 	dh_systemd_start -pubuntu-advantage-tools ua-timer.timer
+	dh_systemd_start -pubuntu-advantage-tools ua-license-check.path
```

Comment:
I'm uncertain we actually want to start this on package install, I'm seeing this result in starting up the license-check.service for a one time run on non gcp systems. Checking to confirm the absence of this will still allow the systemd path to operate properly on google cloud

Reply:
Ok, turns out this symtom was likely due to the fact that I had a stale copy of this branch. @orndorffgrant mentioned at standup that he recently shifted the timer to a oneshot service and reset some of the WantedBy config directives which were causing the license service to run on each boot. I no longer see this behavior on my bionic lxc